packages/e2e-test-utils/src/create-user.js

```diff
@@ -10,14 +10,25 @@ import { switchUserToAdmin } from './switch-user-to-admin';
 import { switchUserToTest } from './switch-user-to-test';
 import { visitAdminPage } from './visit-admin-page';
 
+/**
+ * @typedef {'Subscriber' | 'Contributor' | 'Author' | 'Editor' | 'Administrator'} Role
+ */
+
 /**
  * Create a new user account.
  *
- * @param {string}  username  User name.
- * @param {string?} firstName First name.
- * @param {string?} lastName  Larst name.
+ * @param {string}  username           User name.
+ * @param {Object?} object             Optional Settings for the new user account.
+ * @param {string}  [object.firstName] First name.
+ * @param {string}  [object.lastName]  Last name.
+ * @param {Role}    [object.role]      Role.
+ *
+ * @return {string} Password for the newly created user account
  */
-export async function createUser( username, firstName, lastName ) {
+export async function createUser(
```

Comment:
While this util is so new, it's still been public. I'm wondering if it's okay to change this function's signature or handle the deprecation.

Reply:
Still unreleased on npm though, no?

Reply:
> Still unreleased on npm though, no?

[Yeah](https://www.npmjs.com/package/@wordpress/e2e-test-utils), that was my line of thinking, also.

Reply:
Oh sorry.. If that's the case we are good to go 😄